### ISSUE TITLE
v4.20 - settings window doesn't work

### ISSUE THREAD

**Issue by toxpal** [facelessuser/Rummage#458]
I'm not sure of this is related to https://github.com/facelessuser/Rummage/issues/457 or not, but in v4.20 clicking "File > Preferences" does nothing.

Downgrading to v4.19 solves the issue.

**Comment by facelessuser** [comment]
Weirdly, that may actually be the case. After fixing the preference dialog, the command line was available after install, at least on mac. I'll push the release and see if it fixes it for you.

**Comment by facelessuser** [comment]
@toxpal try updating to 4.20.1 and let me know how it goes.

**Comment by toxpal** [comment]
4.20.1 fixed issue with preferences window, but command line still not available in Linux.

**Comment by facelessuser** [comment]
Okay, this is weird. I uninstalled and installed the latest release via:

```
pip install rummage
```

And the command line broke.

I uninstalled and installed directly from from master and then command line worked:

```
python3 -m pip install -U git+https://github.com/facelessuser/Rummage.git
```

So, something weird is different between installing from the wheel and not from the wheel.

**Comment by toxpal** [comment]
Well, at least we see some progress with debugging here :)

**Comment by facelessuser** [comment]
Yeah, this is good, but it is a weird one though. Installing from the tarball or wheel gives me no command line, but installing from the git repo does give me the command line. 🤔 

Something is different about the installs, but there shouldn't be.

**Comment by toxpal** [comment]
I know you won't be happy about yet another bug, but... :)

After upgrading to 4.20.1, app window title is blank. The same with Rummage entry in taskbar - it's completely blank.

https://i.postimg.cc/sxSywMW7/rummage.png

**Comment by facelessuser** [comment]
😱 That seems to be Linux specific...it's working here. I'll take a look once I resolve this command line nightmare.

**Comment by facelessuser** [comment]
I've created an issue for the title. While I think hatch has a regression that is breaking command line usage, we can work around it with a static define. We will have to temporarily drop `rummage<python-version>` availability. I should be able to look into Linux title issue next. Hopefully, it is an easy fix. GTK is always breaking stuff...I wish it just worked consistently. I haven't changed how titles work in years...
